.docker/Dockerfile

```diff
@@ -13,45 +13,50 @@ RUN apt-get update && \
       gcc \
       git \
       intltool \
-      libglib2.0-dev \
-      libxml2-dev \
-      meson \
+      python3-pip \
+      ninja-build \
       pkg-config \
       unzip \
       wget \
+      libxml2-dev libglib2.0-dev cmake libusb-1.0-0-dev gobject-introspection \
+                 libgtk-3-dev gtk-doc-tools  xsltproc libgstreamer1.0-dev \
+                 libgstreamer-plugins-base1.0-dev libgstreamer-plugins-good1.0-dev \
+                 libgirepository1.0-dev gettext \
     && rm -rf /var/lib/apt/lists/*
 
-RUN wget http://ftp.acc.umu.se/pub/GNOME/sources/aravis/0.6/aravis-0.6.4.tar.xz && \
-    tar xfJ aravis-0.6.4.tar.xz && \
-    rm aravis-0.6.4.tar.xz && \
-    cd aravis-0.6.4 && \
-    ./configure && \
-    make && \
-    checkinstall --pkgname aravis --requires="libglib2.0-dev, libxml2-dev" && \
-    ldconfig
+RUN pip3 install meson
```

Comment:
Run this as part of the run command starting on line 8. as that one installs all dependencies already. Also use the `--no-cache` option to reduce the image size slightly.

Reply:
Fixed